<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTailConfig</class>
 <widget class="QDialog" name="QTailConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="lstSelectOption">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>300</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideMiddle</enum>
       </property>
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="uniformItemSizes">
        <bool>true</bool>
       </property>
       <property name="selectionRectVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tbOptions">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>300</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lstSelectOption</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>QTailConfig</receiver>
   <slot>MenuSelected(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>QTailConfig</receiver>
   <slot>ButtonClicked(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>610</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>MenuSelected(QListWidgetItem*)</slot>
  <slot>ButtonClicked(QAbstractButton*)</slot>
 </slots>
</ui>
